src/pages/demandes/[demandId].tsx

```diff
@@ -0,0 +1,47 @@
+import DemandDetails from '@components/DemandDetails';
+import Manager from '@components/manager';
+import MainContainer from '@components/shared/layout';
+import Slice from '@components/Slice';
+import { GetServerSideProps } from 'next';
+import { getSession } from 'next-auth/react';
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+
+export default function Gestionnaire(): JSX.Element {
+  const router = useRouter();
+  const { demandId }: { demandId?: string } = router.query;
+
+  return (
+    <>
+      <Head>
+        <title>France Chaleur Urbaine - Espace gestionnaire</title>
+      </Head>
+      <MainContainer currentMenu="/demandes">
+        {!demandId ? (
```

Comment:
demandId is always defined here

Reply:
Not if you directly send the URL: `requests/` without requestId
This is an idea on how to organize our URL (`requests/` > for all requests and `requests/[demandId]` for a specific request) but, we can also do a redirect.